lib/vp9.py

```diff
@@ -29,10 +29,12 @@ def __init__(self, name='vp9'):
     super(Vp9Codec, self).__init__(name)
     self.extension = 'webm'
     self.option_set = encoder.OptionSet(
-      encoder.IntegerOption('cpu-used', 0, 16),
+      encoder.IntegerOption('cpu-used', 0, 8),
```

Comment:
They used to use the range 9..16 for experimental speedups. Now I think they use -8..-1 for the same purpose. I've cut off the upper numbers (which will invalidate the parameter sets that use those numbers).